charts/openmetadata/templates/db-migrations-hook-cm.yaml

```diff
@@ -7,7 +7,14 @@ metadata:
     "helm.sh/hook": pre-upgrade 
     "helm.sh/hook-weight": "-10" # we use a smaller weight so it's created before the job  
     "helm.sh/hook-delete-policy": hook-succeeded, hook-failed
+    {{- if .Values.annotations }}
```

Comment:
All the Kubernetes Objects should refer to this by calling
`{{- include "OpenMetadata.annotations" . | nindent 4 }}`.

Refactor to _helper.tpl as mentioned above in review.

Reply:
added {{- include "OpenMetadata.annotations" . | nindent 4 }}